tests/manual-test-cases/Group5-Functional-Tests/5-21-Datastore-Path.robot

```diff
@@ -43,7 +43,7 @@ Setup Suite ESX
     Remove Environment Variable  BRIDGE_NETWORK
     Remove Environment Variable  PUBLIC_NETWORK
 
-    ${output}=  Install VIC Appliance To Test Server  certs=${false}
+    ${output}=  Install VIC Appliance To Test Server  certs=${false}  additional-args=--timeout=%{TEST_TIMEOUT}
```

Comment:
This keyword runs `vic-machine create` with `--timeout=%{TEST_TIMEOUT}`: https://github.com/vmware/vic/blob/master/tests/resources/VCH-Util.robot#L340

We're setting `TEST_TIMEOUT` to 30m on line 39, so if I'm understanding correctly, I don't think we need to change this line.

Reply:
Yep agreed -- I got lost in the `VCH-Util.robot` redirection...and also confused the 2nd & 3rd tests...